pkgs/tools/system/auto-cpufreq/default.nix

```diff
@@ -2,16 +2,21 @@
 
 python3Packages.buildPythonPackage rec {
   pname = "auto-cpufreq";
-  version = "1.7.1";
+  version = "1.9.5";
 
   src = fetchFromGitHub {
     owner = "AdnanHodzic";
     repo = pname;
     rev = "v${version}";
-    sha256 = "1r27ydv258c6pc82za0wq8q8fj0j3r50c8wxc6r7dwr6wx8q3asx";
+    sha256 = "sha256-+8M7PJiJvfFAKYgN93R4NML9GQR989DRnJnv+j0sjUI=";
   };
 
-  propagatedBuildInputs = with python3Packages; [ click distro psutil ];
+  propagatedBuildInputs = with python3Packages; [
```

Comment:
This is just a minor thing but it would be good if you can sort the dependencies alphabetically

Reply:
Happy to; this was the sort order from the upstream requirements.txt